build-support/bin/ci.sh

```diff
@@ -167,11 +167,8 @@ fi
 if [[ "${skip_internal_backends:-false}" == "false" ]]; then
   banner "Running internal backend python tests"
   (
-    targets=$(
-      ./pants.pex list pants-plugins/tests/python:: | \
-      xargs ./pants.pex filter --filter-type=python_tests
-    ) && \
-    ./pants.pex ${PANTS_ARGS[@]} test.pytest ${targets}
+    ./pants.pex ${PANTS_ARGS[@]} test.pytest --compile-python-eval-skip \
```

Comment:
Is `--compile-python-eval-skip` set in all but one shard? If so, could maybe extract to a constant?